.circleci/create_circleci_config.py

```diff
@@ -167,13 +145,11 @@ def to_dict(self):
                 if test.endswith(".py"):
                     expanded_tests.append(test)
                 elif test == "tests/models":
-                    expanded_tests.extend([os.path.join(test, x) for x in os.listdir(test)])
+                    expanded_tests.extend(glob.glob("tests/models/**/test*.py", recursive=True))
```

Comment:
is this to distribute the workload at the python file level ?

Reply:
FYI: I have seen that, once the list being way too long (and therefore pytest get a super long string in the arguments, it will fail

Reply:
Ahhh that a good catch. I might revert.
Same here the splitting can be better if it's on sub files not folders

Reply:
and circle ci suppo9rt that better

Reply:
OK, as long as it works so far, no need to revert. let's see